asynctest/case.py

```diff
@@ -172,13 +172,23 @@ def _init_loop(self):
 
         self.loop = self._patch_loop(self.loop)
 
+    @staticmethod
+    def _scheduled_handle_alive(handle):
+        cancelled_timer_handle = (
+            isinstance(scheduled_thing, asyncio.TimerHandle) and
```

Comment:
Is `scheduled_thing` actually `handle`?
I prefer the name `scheduled_timer_handle_is_alive()`: the functions contains a verb and is more precise (as it returns `False` for `asyncio.Handle`).